metakube/resource_metakube_cluster.go

```diff
@@ -218,6 +220,12 @@ func metakubeResourceClusterCreate(ctx context.Context, d *schema.ResourceData,
 	return metakubeResourceClusterRead(ctx, d, m)
 }
 
+func ensureBillingTenantForAWS(spec *models.ClusterSpec) {
+	if spec.Cloud.Aws != nil {
+		spec.BillingTenant = spec.Cloud.Aws.OpenstackBillingTenant
```

Comment:
Can we move this into metakubeResourceClusterExpandSpec? If you do so it would set BillingTenant on updates as well which I think good.